LiteFarmOrg/LiteFarm
# Lf 2502/hotfix last updated time

To test: 

1. go to http://localhost:3000/sensor/<location_id>/readings
2. check the last updated text format: 

it should look like this: 
<img width="1133" alt="Screen Shot 2022-08-09 at 2 38 21 PM" src="https://user-images.githubusercontent.com/20675885/183766429-036be3e9-ace2-4c5a-87e3-e3b20ea1d8ac.png">
